pages/api/network/management/[networkId]/whitelist/[[...address]].ts

```diff
@@ -36,4 +36,4 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
   res.end();
 }
 
-export default GovernorRoute(handler);
\ No newline at end of file
+export default withProtected(WithValidChainId(withGovernor(handler, ["GET", "DELETE"])));
```

Comment:
why is "delete" an allowed method for _everyone_? This should be available just for governors, no ?